ui/mirage/handlers/mfa.js

```diff
@@ -0,0 +1,146 @@
+import { Response } from 'miragejs';
```

Comment:
@zofskeez do we need the mirage handler now that the API is done, or will this be useful later for any testing or debugging?